spec/models/shop_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+require_relative "../../app//models/shop.rb"
+
+RSpec.describe Shop, :type => :model do
+  it "is not valid without a name, address, city, state" do
```

Comment:
generally we should have separate `it` behaviors for validating each individual field requirement